cpp-client/build-dependencies.sh

```diff
@@ -346,23 +407,38 @@ fi
 
 BUILD_DIR=build_dir
 
-### absl
-if [ "$CLONE_ABSL" = "yes" ]; then
+cmake_common_args="${cmake_pic_arg} ${cmake_shared_arg} -DCMAKE_BUILD_TYPE=${BUILD_TYPE}"
+
+if [ "$shared" = "yes" ]; then
+  LD_LIBRARY_PATH=""
+  for lib in \
+    PROTOBUF RE2 GFLAGS ABSEIL \
```

Comment:
Can this list be put into a variable to avoid repeating it (other appearance is at line 383) ?

Reply:
Done, thanks, good call!